### PR TITLE
Add September 2022 Ecosystem Spotlight

### PR DESCRIPTION
**Pull request by gordoncc**
Added new spotlight which displays properly on /ecosystem, but the old content is still showing up on the homepage index.  @matildepark can you update so that the latest ecosystem spotlight displays on the homepage cards?

### CODE REVIEW

File: content/ecosystem/spotlight/2022-september.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+You can currently submit and search for applications, groups, and content (notebooks and collections), as well as browse by tag.
+
+![Screenshot](https://storage.googleapis.com/media.urbit.org/site/ecosystem/applications/sphinx2.png)
+
+"""
+
+[featured-2]
+title = "peat"
+description = "Need a personal website, fast? Studio can set one up in under a minute."
```
Comment (matildepark) [inline content/ecosystem/spotlight/2022-september.md]: Needs a description

**Comment by gordoncc** [comment]
Looks like we need to account for apps that only use background colors on these cards.
Screen Shot 2022-08-30 at 1 38 49 PM

**Comment by matildepark** [comment]
We do (in Safari) — 
image

Chrome layout engine discrepancy, let's see what the difference is

**Comment by gordoncc** [comment]
Hmm it doesn't look great w/o an image, I should probably just put a screenshot there.

**Comment by gordoncc** [comment]
Ok, a few more issues – there's no background color on the sphinx image in the card, and when I add an image for a card it appears in the spotlight icon thumbnail.  Maybe that was deliberate but we should take that out, then we can use custom images for the index cards.

Screen Shot 2022-08-30 at 2 07 53 PM

**Comment by matildepark** [comment]
Can you push?

**Comment by gordoncc** [comment]
Pushed

**Comment by gordoncc** [comment]
Looking much better, except adding a custom image replaces the post image now ... anyway we can get that original podcast/app image in there and keep the custom image on the index card?

Screen Shot 2022-08-30 at 4 10 16 PM

**Comment by matildepark** [comment]
That's how it was intended to be written but I dropped one. Should be fine now.

**Comment by gordoncc** [comment]
lgtm

**Review by matildepark — APPROVED** [review]

**Comment by matildepark** [comment]
Merge or hold until Sep?

**Comment by gordoncc** [comment]
It's called we release just a little ahead of schedule, like magazines

**Pull request merged**